### DISCUSSION TITLE
安装上用配置什么吗？

### DISCUSSION THREAD

**Discussion by elarbor** [fhefh2015/Fast-GitHub#85] [General]
这样是不是就算成功了？
![image](https://github.com/user-attachments/assets/65b69df1-bc99-483c-8595-d5a7d40ab899)

![image](https://github.com/user-attachments/assets/f1cd8575-3b27-4260-a5bf-41d5f6aa626e)

**Comment by HuangYan-lz** [comment]
您好，邮件已经收到，我将尽快给你回复。
